### ISSUE TITLE
Making https://julialang.org conform to WebAim Accessibility standards

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaLang/www.julialang.org#458]
https://wave.webaim.org/report#/https://julialang.org

It seems like there's a lot of low hanging fruit that we can take care of. I can try and open up Pull requests for some of these, though my web dev experience is lackluster. 

Also planning to make a bot that will open similar issues on documentation/packaged websites.

**Comment by logankilpatrick** [comment]
The first update is made! The number of errors is down from 16 to 15!

**Comment by logankilpatrick** [comment]
Once these issues are resolved, we can get the Julia website certified https://webaim.org/services/certification/

**Comment by logankilpatrick** [comment]
Only one alert left to fix related to a linked PDF. I reached out to WAVE for suggestions as to how to fix it. Other than that, all the errors and alerts have been fixed!!! I also made an `#accessibility` channel on Slack to keep up the dialog on similar threads to this one.

**Comment by ViralBShah** [comment]
Maybe just use the `#website` channel for all website related stuff? Except `#seo`...
